### PR TITLE
order fe + be

### PR DESCRIPTION
**Pull request by dungpt-0371**
## Related Tickets:
- [#68088](https://edu-redmine.sun-asterisk.vn/issues/68088)
- [#68092](https://edu-redmine.sun-asterisk.vn/issues/68092)

## WHAT this PR does?
- Giao diện đặt hàng
- Be xử lí tạo đơn hàng mới

## Notes Impacted Areas:
- app/Http/Controllers/CartController
- app/Http/Controllers/OrderController
- resources/views/cart/index.blade.php
- routes/web.php

## Attachments:
image

### CODE REVIEW

**Comment by dungpt-0371** [comment]
ready

**Comment by hieupc-0248** [comment]
oke

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+        $order->save();
+
+        $carts = Cart::where('user_id', $user->id)->get();
+
+
+        foreach ($carts as $book) {
+            $book->delete();
+        }
+        return redirect('/carts');
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]:
```suggestion
        DB::table('carts')->whereIn('id', $carts->pluck('id'))->delete();

        return redirect('/carts');
```
giảm số query, add space trc return

File: resources/views/cart/index.blade.php [hunk 2]
```diff
@@ -4,117 +4,108 @@
...
+                            </th>
+                            <th class="relative px-6 py-3 text-gray-500 dark:text-red-100">
+                                <span class="sr-only uppercase"> {{ __("Delete") }}
+                                </span>
+                            </th>
+                        </tr>
+                    </thead>
+                    <tbody class="bg-slate-50 divide-y divide-gray-200 dark:divide-gray-600">
+                        @foreach ($cartItems as $cartItem)
+                        <tr class="cart-item-wrap">
```
Comment (ghost) [inline resources/views/cart/index.blade.php]: indent 4 spaces

**Review by ghost — COMMENTED** [review]
ignore public css, js...

**Comment by dungpt-0371** [comment]
ready

**Pull request merged**